packages/core/addon/models/dashboard-widget.ts

```diff
@@ -41,7 +41,7 @@ export default class DashboardWidget extends Model.extend(hasVisualization, Vali
    */
   @computed('dashboard.owner')
   get owner() {
-    return this.dashboard.owner;
+    return this.dashboard.get('owner');
```

Comment:
required?

Reply:
Yep, without this I'm now getting because I've added the Asset owner component to the widget edit/view route
```
Error: Assertion Failed: You attempted to access the `owner` property (of <(unknown):ember7937>).
Since Ember 3.1, this is usually fine as you no longer need to use `.get()`
to access computed properties. However, in this case, the object in question
is a special kind of Ember object (a proxy). Therefore, it is still necessary
to use `.get('owner')` in this case.
```

I've updated the type and now the existing code shows up as an error